doc/md/tutorial-todo-crud.md

```diff
@@ -50,6 +50,16 @@ func (Todo) Fields() []ent.Field {
 }
 ```
 
+Update your imports to look like this:
+```
+import (
+	"time"
+
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+)
+```
+
```

Comment:
Can you edit the example above instead?

Reply:
It would not make sense because the example is only a snippet. It is missing the struct defintion, so adding the imports on top would be confusing I think

Reply:
Most users use `goimports`, but if you still think it's necessary, let's keep it as one code block. Thanks 🙏 
~~~
```go title="ent/schema/todo.go"
package schema

import (
	"time"
	"entgo.io/ent"
	"entgo.io/ent/schema/field"
)

 // ...

func (Todo) Fields() []ent.Field {
	// ...
}
```
~~~

Reply:
:+1: I am quite new to go and didn't know it can auto import so I got confused by it

Reply:
Checkout https://pkg.go.dev/golang.org/x/tools/cmd/goimports and let me know if we can close this one. Thanks

Reply:
yeah I think it can be closed then

Reply:
Thanks 🙏🏻